scvi/model/_scvi.py

```diff
@@ -258,9 +257,10 @@ def _get_latent_fields(mode: LatentDataType):
 
     def to_latent_mode(
```

Comment:
Here is what I might do: Keep the contents of this function as it is today (per the main branch). Add an option in this call to return a copy of the adata. If the user asks for that option, then return make and return a copy of the adata otherwise leave everything as is (i.e. override the data). I believe this would address my comment below as well (the comment regarding whether the copy is making any difference the way it is currently implemented).